thedmstoolkit/character_generation.py

```diff
@@ -0,0 +1,538 @@
+import random
+from typing import Callable, Optional, Union
+
+from toolkit.models import Character
+
+Generator = Callable[[int, int], int]
+
+
+class Character_Generator:
+    """
+    Contains all the lists and Dictionaries
+    for the various character generation methods
+    """
+
+    RACE_DICT: dict[str, list] = {
```

Comment:
should these dicts go into the database?

Reply:
I don't know I think this is fine for now